src/main/java/ch/njol/skript/expressions/ExprPotionEffect.java

```diff
@@ -20,16 +21,20 @@
 @Name("Potion Effect")
 @Description({"Create a new potion effect to apply to an entity or item type. Do note that when applying potion effects ",
 	"to tipped arrows/lingering potions, Minecraft reduces the timespan."})
-@Examples({"set {_p} to potion effect of speed of tier 1 without particles for 10 minutes",
+@Examples({
+	"set {_p} to potion effect of speed of tier 1 without particles for 10 minutes",
+	"set {_p} to infinite potion effect of darkness of tier 1 without particles",
+	"set {_p} to ambient potion effect of darkness of tier 1 without particles without the icon for 3 minutes",
 	"add {_p} to potion effects of player's tool",
 	"add {_p} to potion effects of target entity",
-	"add potion effect of speed 1 to potion effects of player"})
-@Since("2.5.2")
+	"add potion effect of speed 1 to potion effects of player"
+})
+@Since({"2.5.2", "INSERT VERSION (infinite/no icon)"})
 public class ExprPotionEffect extends SimpleExpression<PotionEffect> {
 	static {
 		Skript.registerExpression(ExprPotionEffect.class, PotionEffect.class, ExpressionType.COMBINED,
-			"[new] potion effect of %potioneffecttype% [potion] [[[of] tier] %-number%] [(1¦without particles)] [for %-timespan%]",
-			"[new] ambient potion effect of %potioneffecttype% [potion] [[[of] tier] %-number%] [(1¦without particles)] [for %-timespan%]");
+			"[new] [:ambient] potion effect of %potioneffecttype% [potion] [[[of] tier] %-number%] [noparticles:without [any] particles] [icon:(whilst hiding [the]|without (the|a)) [potion] icon] [for %-timespan%]",
+			"[new] infinite [:ambient] potion effect of %potioneffecttype% [potion] [[[of] tier] %-number%] [noparticles:without [any] particles] [icon:(whilst hiding [the]|without (the|a)) [potion] icon]");
```

Comment:
```suggestion
			"[new] [:ambient] potion effect of %potioneffecttype% [potion] [[[of] tier] %-number%] [noparticles:without [any] particles] [icon:without (the|a) [potion] icon] [for %-timespan%]",
			"[new] infinite [:ambient] potion effect of %potioneffecttype% [potion] [[[of] tier] %-number%] [noparticles:without [any] particles] [icon:without (the|a) [potion] icon]");
```
makes the syntax a little easier to read, `whilst` is not very common nowadays

Reply:
While I do agree, this was copy/pasted directly from EffPotion, the reason for that was parity.

Reply:
ahh screw it, lets live a little 😂